plugins/@grouparoo/postgres/src/lib/connect.ts

```diff
@@ -43,6 +44,18 @@ export const connect: ConnectPluginAppMethod = async ({ appOptions }) => {
     }
   });
 
+  // Unfortunately there's no event listener for queries so we have to intercept the call.
+  // https://github.com/brianc/node-postgres/issues/2580
+  const shim: typeof pool.query = pool.query.bind(pool);
+  (pool as any).query = (
+    query: Parameters<typeof pool.query>[0],
+    data?: Parameters<typeof pool.query>[1],
+    callback?: Parameters<typeof pool.query>[2]
+  ) => {
+    log(`[ postgres ] ${query}`, "debug", data);
+    return shim(query, data, callback);
```

Comment:
These might be a little more resilient with sending `(*arguments)` or something like that and only opting into the ones needed for logging. But maybe just something to look out for if there is churn.

Reply:
I'm intentionally doing this so if the method signature changes we'll need to update this as well.
While unlikely, the first argument could be changed to something else in an update.